.github/workflows/bundle-size.yml

```diff
@@ -14,5 +14,5 @@ jobs:
     - uses: preactjs/compressed-size-action@v1
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
-        build-script: "npx lerna run build:module"
+        build-script: "bundlesize"
```

Comment:
Does this mean that this check will fail if the size exceeds the limit?

Reply:
I am going to increase the size for now and allow no failures to make sure we catch bugs like this.

Reply:
We cannot use this as you said, fixed it.

Reply:
Increased the size limit for now to avoid it from breaking.